src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -261,6 +263,13 @@ public async Task Batch()
 
                 var counterBatch = JsonDeserializationClient.CounterBatch(countersBlittable);
 
+                if (TrafficWatchManager.HasRegisteredClients)
+                {
+                    var sb = new StringBuilder();
+                    sb.Append(/*"Counter:\n"+ */countersBlittable);
+                    HttpContext.Items["TrafficWatch"] = sb.ToString();
```

Comment:
Please create a method that would accept a string only and set this internally. Avoid having the string constant repeated

Reply:
I can create a method wich accepts a string as argument, in each class that we are taking the customInfo from.
Or I can create a method inside the trafficWatchManager and pass the httpContext with string to it.

Reply:
All the handlers inherit from a common base, put that there.

Reply:
done